src/adapters/iosAdapter.ts

```diff
@@ -143,12 +143,14 @@ export class IOSAdapter extends AdapterCollection {
         debug(`iOSAdapter.getProxyPath`)
         return new Promise((resolve, reject) => {
             if (os.platform() === 'win32') {
-                const proxy = os.arch() === 'x64' ? path.resolve(__dirname, '../../node_modules/vs-libimobile/x64/ios_webkit_debug_proxy.exe') : path.resolve(__dirname, '../../node_modules/vs-libimobile/lib/ios_webkit_debug_proxy.exe');
+                const x64 = os.arch() === 'x64';
+                const proxy = x64 ? path.resolve(__dirname, '../../../../../../../scoop/apps/ios-webkit-debug-proxy/current/ios_webkit_debug_proxy.exe') : path.resolve(__dirname, '../../node_modules/vs-libimobile/lib/ios_webkit_debug_proxy.exe');
```

Comment:
This path doesn't seem right. See https://github.com/lukesampson/scoop/wiki/Global-Installs